lib/graide/glyph.py

```diff
@@ -50,9 +50,8 @@ class GlyphItem(object) :
     def __init__(self, face, gid, height = 40) :
         face.set_char_size(height = int(height * 64))
         (self.pixmap, self.left, self.top) = ftGlyph(face, gid)
-        n = ctypes.create_string_buffer(64)
-        freetype.FT_Get_Glyph_Name(face._FT_Face, gid, n, ctypes.sizeof(n))
-        self.name = re.sub(ur'[^A-Za-z0-9._]', '', n.value) # Postscript name
+        name = face.get_glyph_name(gid).decode('ascii')
+        self.name = re.sub('[^A-Za-z0-9._]', '', name) # Postscript name
```

Comment:
Where is the implementation of get_glyph_name() and load_glyph()

Reply:
https://github.com/rougier/freetype-py/blob/master/freetype/__init__.py